app/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx

```diff
@@ -40,7 +41,24 @@ export const BlocksInfo: React.FC<BlockInfoProps> = ({ block }) => {
     <Card>
       <Header>
         <IconH2Container>
-          {themeType === 'light' ? <BlocksIconLight /> : <BlocksIconDark />}
+          {themeType === 'light' ? (
```

Comment:
It seems like you could rewrite this more concisely like: 

```
   <Icon
              icon="BlocksIcon"
              width={49}
              height={16}
              color="none"
              stroke={themeType === 'light' ? colors.black : colors.white}
            />
```

Reply:
Clean! Updated to this format